src/editor.html

```diff
@@ -1696,6 +1710,10 @@
 				padding-left: 10px;
 				padding-right: 10px;
 			}
+
+			#zss_editor_footer {
+				height: 10px;
```

Comment:
I used footer as a default gap between last line in editor and keyboard. It gives a better UI. I use its height in updateOffset function. DYT we should specify its height via props?